### PR TITLE
Allow custom messages for health checks in /api/health

### PR DESCRIPTION
**Pull request by jessemyers**

### CODE REVIEW

**Comment by afallou** [comment]
+1

File: microcosm_flask/tests/conventions/test_health.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def test_health_check_custom_check():
         "ok": True,
         "checks": {
             "foo": {
-                "message": "ok",
```
Comment (cdm-ium) [inline microcosm_flask/tests/conventions/test_health.py]: This actually surprises me a little, isn't the default result `ok`?

**Comment by cdm-ium** [comment]
+1

File: microcosm_flask/tests/conventions/test_health.py (hunk 1 above)
Comment (jessemyers) [inline microcosm_flask/tests/conventions/test_health.py]: See L45

**Pull request closed without merging**